docker-compose.yml

```diff
@@ -55,4 +50,4 @@ services:
 volumes: {}
 
 networks:
-  default:
\ No newline at end of file
+  default:
```

Comment:
It's 2021; what editors are still not properly adding newlines to the end of files. :|

Reply:
I see this a lot, so I'm going to leave this stackoverflow answer as a reference for why that last newline is important: https://stackoverflow.com/a/729795

Reply:
VSCode! I've enabled the option to add the line, so it shouldn't happen again.

Thanks for the link as well!